src/migrations/201602231400_initial.js

```diff
@@ -17,7 +17,7 @@ function createAccountsTable (knex) {
 function createFulfillmentsTable (knex) {
   return knex.schema.createTableIfNotExists('fulfillments', (table) => {
     table.increments()
-    table.uuid('transfer_id').unique().index()
+    table.uuid('transfer_id').index()
```

Comment:
Yeah, it looks like this gets the database into the same "stuck state" that I was in. I was also able to get rid of the failures by dropping all tables. Hopefully we can figure this out at some point, but it looks like it might not be so easy, so we don't have to block on this.